asterisk/rootfs/etc/cont-init.d/asterisk.sh

```diff
@@ -176,6 +176,17 @@ if bashio::var.false "$(bashio::config 'mailbox')"; then
     touch /tmp/disable-asterisk-mailbox
 fi
 
+# Enable voicemail persistence
+media_voicemail_dir="/media/asterisk/voicemail"
+media_voicemail_extension_dir="${media_voicemail_dir}/default/$(bashio::config 'mailbox_extension')/"
+asterisk_dir="/var/spool/asterisk"
+asterisk_voicemail_dir="${asterisk_dir}/voicemail"
+
+mkdir -p "${media_voicemail_extension_dir}/"{"INBOX","Old","Urgent"}
+
+rm -rf "${asterisk_voicemail_dir}"
+ln -s "${media_voicemail_dir}" "${asterisk_voicemail_dir}"
```

Comment:
I'm just wondering, can't we change the path through the Asterisk configuration files?